library/include/hipblas.h

```diff
@@ -100,18 +100,7 @@
 /*! \brief hipblasHanlde_t is a void pointer, to store the library context (either rocBLAS or cuBLAS)*/
 typedef void* hipblasHandle_t;
 
-/*! \brief To specify the datatype to be unsigned short */
-
-#if __cplusplus < 201103L || !defined(HIPBLAS_USE_HIP_HALF)
-
-typedef uint16_t hipblasHalf;
-
-#else
-
-#include <hip/hip_fp16.h>
-typedef __half hipblasHalf;
-
-#endif
+typedef _Float16 hipblasHalf;
```

Comment:
It isn't really part of c++11 standard it is an extension from what I read.
I would keep the 
#if _cplusplus < 201103L 
as for API exposure with C/early C++ the uint16_t should suffice as before?